CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ All notable changes to the [Camunda Modeler](https://github.com/camunda/camunda-
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.0.0-alpha.1
+
+#### General
+
+* `FEAT`: new loading indicator ([#1719](https://github.com/camunda/camunda-modeler/issues/1719))
+* `FEAT`: new validation logic for deployment diagram ([#1709](https://github.com/camunda/camunda-modeler/issues/1709))
+* `FEAT`: save and restore window size ([#576](https://github.com/camunda/camunda-modeler/issues/576))
+* `FEAT`: apply native keyboard shortcuts in log panel ([#1380](https://github.com/camunda/camunda-modeler/issues/1380))
+* `CHORE`: dialogs to return response instead of button ([`42d0ecc`](https://github.com/camunda/camunda-modeler/commit/42d0eccfae6143f472ec60b0280c7267b7d5a4e5))
+* `CHORE`: remove watermark ([#1731](https://github.com/camunda/camunda-modeler/issues/1731))
+* `CHORE`: bump to `dmn-js@8.2.0`
+
+#### BPMN
+
+* `FIX`: save bpmndi as drawn on the diagram ([#985](https://github.com/bpmn-io/bpmn-js/issues/985))
```

Comment:
This was actually already fixed via `[3.7.1](https://github.com/camunda/camunda-modeler/blob/v3.7.1/client/package.json#L12)`.